src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -96,11 +96,18 @@ public void setDashboards(final OpenshiftRequestContext context, Client client,
         // compare results to projects; handle any deltas (create, delete?)
 
         Set<String> indexPatterns = getProjectNamesFromIndexes(context, client, projectPrefix);
-        LOGGER.debug("Found '{}' Index patterns for user", indexPatterns.size());
+        LOGGER.debug("Found '{}' Index patterns for user {}", indexPatterns.size(), indexPatterns);
 
         Set<String> projects = new HashSet<>(context.getProjects());
         if(context.isOperationsUser()) {
             projects.add(OPERATIONS_PROJECT);
+            for (String pattern : indexPatterns) {
+                // Project might no longer exist, but the data can still be retrieved from elasticsearch
+                if (!projects.contains(pattern) && !pattern.equals(ADMIN_ALIAS_NAME)) {
```

Comment:
I don't believe alias are returned as part of the listed indices which I would think its then not necessary to compare against those